Makefile

```diff
@@ -25,7 +25,7 @@ push-tag:
 
 .PHONY: install-otel-builder
 install-otel-builder:
-	GO111module=on go install go.opentelemetry.io/collector/cmd/builder@v0.69.1
+	GO111module=on go install go.opentelemetry.io/collector/cmd/builder@v0.78.1
```

Comment:
the latest release for core repo components is 0.78.2

Reply:
It's ok. This is just a dependency, and this module doesn't depend on the batch processor. 
I've got otel-launcher-go, otel-arrow-adapter, experimental-arrow-collector, this repo, and lightstep internal to think about, and I've got them all at 0.78.1.